python/lsst/pipe/base/tests/mocks/_pipeline_task.py

```diff
@@ -121,6 +124,53 @@ def mock_task_defs(
     return results
 
 
+def mock_pipeline_graph(
+    original_graph: PipelineGraph,
+    unmocked_dataset_types: Iterable[str] = (),
+    force_failures: Mapping[str, tuple[str, type[Exception] | None]] | None = None,
+) -> PipelineGraph:
+    """Create mocks for a full pipeline graph.
+
+    Parameters
+    ----------
+    original_graph : `~..pipeline_graph.PipelineGraph`
+        Original tasks and configuration to mock.
+    unmocked_dataset_types : `~collections.abc.Iterable` [ `str` ], optional
+        Names of overall-input dataset types that should not be replaced with
+        mocks.
+    force_failures : `~collections.abc.Mapping` [ `str`, `tuple` [ `str`, \
+            `type` [ `Exception` ] or `None` ] ]
+        Mapping from original task label to a 2-tuple indicating that some
+        quanta should raise an exception when executed.  The first entry is a
+        data ID match using the butler expression language (i.e. a string of
+        the sort passed ass the ``where`` argument to butler query methods),
+        while the second is the type of exception to raise when the quantum
+        data ID matches the expression.
```

Comment:
Had to remind myself since I just copied this from `mock_task_defs`, but the answer is that it's just a way to make it easy to invoke the default of `ValueError`, which is defined down in the `MockPipelineTaskConfig`.  Will document.